service_thing_aggregate/src/it/scala/ObjectSearchIntegrationSpec.scala

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by jarle on 05.10.16.
+ */
+import com.google.inject.Inject
+import dao.ObjectSearchDao
+import models.{MuseumIdentifier, MusitThing, ObjectAggregation, ObjectId}
+import no.uio.musit.test.MusitSpecWithServerPerSuite
+import org.scalatest.time.{Millis, Seconds, Span}
+import play.api.libs.json.Json
+import play.utils.UriEncoding
+
+import scala.language.postfixOps
```

Comment:
Have removed postfixOps.
(No idea where it came from, probable copy+paste from somewhere else)